flink-fault-tolerance/data/expected_output.txt

```diff
@@ -0,0 +1,11 @@
+{"user_id":1,"window_time":1640966459999,"total_item_last_two_minutes":1}
```

Comment:
Where is this file used to verify the output results?

Reply:
I updated the script to check the output results.